pom.xml

```diff
@@ -243,10 +248,37 @@
             <portName>test.harvester.port</portName>
             <portName>test.db.port</portName>
             <portName>test.s3.port</portName>
+            <portName>test.provider.port</portName>
           </portNames>
         </configuration>
       </plugin>
 
+      <!-- Points the test OAI-PMH records at the thumbnails S3 bucket -->
+      <plugin>
+        <groupId>com.google.code.maven-replacer-plugin</groupId>
+        <artifactId>replacer</artifactId>
+        <version>${replacer.plugin.version}</version>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>replace</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <includes>
+            <include>${project.basedir}/target/test-classes/provider/**/*.xml</include>
+          </includes>
+          <replacements>
+            <replacement>
+              <token>S3_BASE_URL</token>
```

Comment:
Was there a reason this couldn't be done using the Maven resources plugin's filter mechanism (using the testResources config it also works on test resources)? It looks like these files are already in the test resources directory. Why use a different/new plugin for this?

Reply:
Ah, I forgot that was possible. Will change.